metadata.json

```diff
@@ -31,6 +31,7 @@
       "operatingsystemrelease": [
         "16.04",
         "18.04"
+        "20.04"
```

Comment:
You also need to modify ```.sync.yml``` and ```.travis.yml```. Please keep in mind that Puppet Inc. does not provide any Puppet 5 packages for Ubuntu 20.04.

Reply:
Ack - I had done a "grep -r 18.04" to look for what I needed to change (this is my first merge request in a long, long, long time :).  I didn't think to do a "grep -r 1804".  :).  I'll get it updated - thanks.

Reply:
I've resubmitted with the updated .sync.yml and .travis.yml.  I see a "Resolve conversation" button - I'm going to guess that if it'll let me hit that button, I'm supposed to hit that button once I've made the change you requested?  :).